geode-core/src/main/java/org/apache/geode/management/internal/ManagementAgent.java

```diff
@@ -546,6 +550,10 @@ public JMXConnectorServer getJmxConnectorServer() {
     return jmxConnectorServer;
   }
 
+  public RemoteStreamExporter getRemoteStreamExporter() {
+    return new GeodeRemoteStreamExporter(port, rmiClientSocketFactory, rmiServerSocketFactory);
```

Comment:
do we need to create a new exporter every time we get it? Seems like RemoteStreamExporter.getInstance() will return the same instance every time.